docs/quickstarts/expo.mdx

```diff
@@ -674,7 +674,15 @@ export default function UseUserExample() {
 ```
 </Steps>
 
-### Next steps
+## Additional consideration
+
+We recommend integrating over the air (OTA) updates into your Expo application. This will enable 
+you to easily roll out feature updates and security patches alike, as they’re applied to Clerk's SDK’s,
+without having to do a full AppStore/GooglePlay review cycle. Check out [`expo-updates`](https://docs.expo.dev/versions/latest/sdk/updates)
+for how this can accomplished.
+
+
+## Next steps
```

Comment:
Intentionally bumping "Next steps" from H3 to H2. This should look better in the righthand sidebar